helm/scylla-operator/templates/scylladbmonitoring_grafana_clusterrole_def_openshift.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scylladb:aggregate-to-scylladb-monitoring-grafana-openshift
+  labels:
+    rbac.operator.scylladb.com/aggregate-to-scylladb-monitoring-grafana: "true"
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - privileged
```

Comment:
grafana doesn't run in privileged mode though?

Reply:
it runs as specific (not random) user - which is privileged

Reply:
Does it have to?

Reply:
yep, it comes from storage permissions as I recall

Reply:
(the image has the user baked in)